test/api/junction_detection_methods_test.cpp

```diff
@@ -56,28 +43,29 @@ TEST(junction_detection, single_method_only)
         seqan3::debug_stream << "-----------------------------------------------------------------------\n"
                              << "Test Method: " << method << '\n'
                              << "-----------------------------------------------------------------------\n";
-        detect_junctions_in_alignment_file(DATADIR"simulated.minimap2.hg19.coordsorted_cutoff.sam",
+        detect_variants_in_alignment_file(DATADIR"simulated.minimap2.hg19.coordsorted_cutoff.sam",
                                            tmp_dir/"detect_breakends_out_short.fasta",
                                            {method},
                                            simple_clustering,
                                            no_refinement,
-                                           sv_default_length);
+                                           sv_default_length,
+                                           empty_path);
 
         if (method == 0)
         {
-            check_output_and_cleanup(expected_res_cigar);
```

Comment:
Instead of replacing the line, could you (comment out the previous one and) add a todo (David) so we can see that something still needs to happen here and who added the todo?